htdocs/install/page_dbsettings.php

```diff
@@ -347,9 +347,9 @@ function xoFormFieldCharset($name, $value, $label, $help = '', $link) {
 	<script type="text/javascript">
 		function setFormFieldCollation(id, val) {
 			if (val == '') {
-				$(id).style.display='display';
```

Comment:
If I understand it correctly, this class should not be 'display', but it should be 'none'?

Reply:
Yes, this was fixed. I think you are looking into old version of code.